build/Targets/Imports.targets

```diff
@@ -353,6 +369,7 @@
 
   <Import Project="GenerateAssemblyInfo.targets" Condition="'$(ProjectLanguage)' == 'CSharp' OR '$(ProjectLanguage)' == 'VB'" />
   <Import Project="GenerateInternalsVisibleTo.targets" />
+  <Import Project="DisableTransitiveReferences.targets" Condition="'$(RoslynProjectType)' == 'Vsix' AND '$(RoslynSdkProject)' == 'true'" />
```

Comment:
Why the conditioning on Vsix projects here? I imagine much profanity in your direction once somebody else gets bit by this...